tcms/templates/run/report.html

```diff
@@ -143,14 +143,6 @@ <h4 style='padding:0px;margin:0px'>Comments</h4>
 				<td><a href="{{ bug_url }}" target="_blank">{{ bug_url }}</a></td>
 			</tr>
 			{% endfor %}
-			<tr>
-				<td>View all bugs(Bugzilla)</td>
-				<td><a href="{{ bugzilla_url }}" target="_blank">{{ bugzilla_url }}</a></td>
-			</tr>
-			<tr>
-				<td>View all bugs(JIRA)</td>
-				<td><a href="{{ jira_url }}" target="_blank">{{ jira_url }}</a></td>
-		  </tr>
```

Comment:
This is part of #185. Given that I seem to be the most active user of Nitrate currently I think we can live this this for the next week or so until I implement it.